transifex/native/django/tools/migrations/templatetags.py

```diff
@@ -280,7 +280,19 @@ def _parse_token(self, token, parser, original_string):
 
         # A variable was found; copy as is
         elif token.token_type == TOKEN_VAR:
-            return StringMigration(original_string, original_string), None
+            # that's a special case we need to take care of:
+            # {{ _("Are you sure you want to remove the ($(collaborator_count)) selected collaborators?")|escapejs }}
+            if token.contents.startswith('_('):
+                token.token_type = TOKEN_BLOCK
+                clos_par_pos = 0
+                for i, j in enumerate(token.contents):
+                    if j == ')':
+                        clos_par_pos = i
```

Comment:
Why not `clos_par_pos = token.contents.index(')')`?

Reply:
@kbairak we need the last one. I could do `len(token.contents) - token.contents[::-1]].index(')')` but this should be difficult to follow ...

Reply:
ah ok